db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, peep VARCHAR(280), time TIMESTAMP);
```

Comment:
This TIMESTAMP business seems pretty handy. Is it a straightforward way of getting the time on the peeps?